architecture/sync.md

```diff
@@ -190,15 +178,13 @@ Verification is performed as follows:
 
 1. The cached access token is examined; access token is missing or expired, an attempt is made to refresh it.
 2. A request is made for the user's profile information:
-
   - If this request fails with an HTTP error code of **403 (Unauthorized)**, an attempt is made to refresh it.
   - If this request succeeds, the user's authorization is valid.
 
 Expired, invalidated, or missing access token can be refreshed as follows:
 
 1. The cached refresh token is examined; if there is no refresh token, any existing access token is cleared from device cache and verification fails with a `AUTH` error.
-2. A POST HTTPS request is made to the FxA OAuth token endpoint, with a grant_type of "refresh_token" and including the cached refresh token:
-
+2. A `POST` HTTPS request is made to the FxA OAuth token endpoint, with a `grant_type` of "refresh_token" and including the cached refresh token:
```

Comment:
Below (L199), we say _"An HTTP "GET" request"_. Not sure if we want to change this to:

> _"An HTTPS "POST" request is made ..."_

or change L199 below to

> _"A `GET` HTTPS request is made ..."_

Reply:
they should be consistent, but I don't have a preference with which way to be consistent.